sksurv/metrics.py

```diff
@@ -89,7 +89,6 @@ def _check_estimate_2d(estimate, test_time, time_points, estimator):
 def _get_comparable(event_indicator, event_time, order):
```

Comment:
Can you please rename `_get_comparable` to `_iter_comparable` to make it clearer what the function does.